lib/cli/commands/listpeers.ts

```diff
@@ -1,11 +1,55 @@
 import { callback, loadXudClient } from '../command';
 import { Arguments } from 'yargs';
-import { ListPeersRequest } from '../../proto/xudrpc_pb';
+import Table, { HorizontalTable } from 'cli-table3';
+import colors from 'colors/safe';
+import { ListPeersRequest, ListPeersResponse } from '../../proto/xudrpc_pb';
+
+const HEADERS = [
+  colors.red('address'),
+  colors.red('node public key'),
+  colors.red('lnd public keys'),
+  colors.red('inbound'),
+  colors.red('pairs list'),
+  colors.red('xud version'),
+  colors.red('seconds connected'),
+  colors.red('raiden address'),
+];
+
+const createTable = () => {
+  const table = new Table({
+    head: HEADERS,
+  }) as HorizontalTable;
+  return table;
+};
+
+const formatPeers = (peers: ListPeersResponse.AsObject) => {
+  const formattedPeers: string[][] = [];
+  peers.peersList.forEach((peer: any) => {
+    const peerInfo: string[] = [];
+    Object.keys(peer).forEach((key) => {
+      if (key === 'pairsList') {
+        colors.green(peer[key]);
```

Comment:
Is there a reason pairslist is being made green? It's not getting pushed to the array in this case, which I believe is the cause of the off by one error karl mentions above.